code/lib/cli-storybook/src/sandbox.ts

```diff
@@ -197,13 +197,9 @@ export const sandbox = async ({
     logger.log(`📦 Downloading sandbox template (${picocolors.bold(downloadType)})...`);
     try {
       // Download the sandbox based on subfolder "after-storybook" and selected branch
-      const gitPath = `github:storybookjs/sandboxes/${templateId}/${downloadType}#${branch}`;
+      const gitPath = `storybookjs/sandboxes/tree/${branch}/${templateId}/${downloadType}`;
       // create templateDestination first (because it errors on Windows if it doesn't exist)
-      await mkdir(templateDestination, { recursive: true });
-      await downloadTemplate(gitPath, {
-        force: true,
-        dir: templateDestination,
-      });
+      spawnSync('npx', ['gitpick', gitPath, templateDestination, '-o']);
```

Comment:
I would feel safer is this was a pinned-down version of gitpick. Supply chain attacks are a thing...
Also, this is a hidden dependency, not made explicit though package.json or node_modules. I don't love that.

Reply:
I see we had `unjs/giget` before (you removed it). Why not use that instead of gitpick? They seem to do the same thing.